ci/ray_ci/automation/docker_tags_lib.py

```diff
@@ -97,6 +99,38 @@ def list_image_tag_suffixes(
     return tag_suffixes
 
 
+def pull_image(image_name: str) -> None:
+    """
+    Pull image from Docker Hub.
+    """
+    client = docker.from_env()
+    client.images.pull(image_name)
+
+
+def get_ray_commit(image_name: str) -> str:
+    """
+    Get the commit hash of Ray in the image.
+    """
+    client = docker.from_env()
+    pull_image(image_name)
+    # Command to grab commit hash from ray image
+    command = "python -u -c 'import ray; print(ray.__commit__)'"
+
+    container = client.containers.run(
+        image=image_name, command=command, remove=True, stdout=True, stderr=True
+    )
+    output = container.decode("utf-8").strip()
+    match = re.search(
+        r"^[a-f0-9]{40}$", output, re.MULTILINE
+    )  # Grab commit hash from output
```

Comment:
why do we need this matching?

we will immediately check if this is the same as `git rev-parse` right?

Reply:
This is to filter and grab only the commit hash. I've seen cases where calling `import ray` prints out a bunch of random stuff about CUDA besides the commit hash and it got into the output.